packages/react-components/src/utils.ts

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { renderToStaticMarkup } from 'react-dom/server';
-import { format } from 'date-fns';
+import { format, utcToZonedTime } from 'date-fns-tz';
```

Comment:
Not sure we should bother with the polyfil. Its apparently got 95% browser support, thoughts?

Reply:
Polyfill what, Intl? nah looks good enough I agree